sdk/storage/storage-blob/src/PageBlobRangeResponse.ts

```diff
@@ -0,0 +1,86 @@
+import { HttpResponse } from "@azure/core-http";
+import * as Models from "./generated/src/models";
+import { Range } from "./Range";
+
+export interface PageList {
+  pageRange?: Range[];
+  clearRange?: Range[];
+}
+
+export interface PageBlobGetPageRangesResponse
```

Comment:
Is it missed to export types here except rangeResponseFromModel from PageBlobRangeResponse through index.browser.ts and index.ts?

Reply:
is there a need to export?
/cc @jeremymeng

Reply:
I'll export the types, thanks for pointing it out :)